File: TEST_ANDROID_VER/app/src/main/java/org/androidtown/tripdiary/MainActivity.java
package org.androidtown.tripdiary;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;

public class MainActivity extends AppCompatActivity {

        ImageButton buttonMenu1;
        ImageButton buttonMenu2;
        ImageButton buttonMenu3;

        @Override
        protected void onCreate(Bundle savedInstanceState) {
            super.onCreate(savedInstanceState);
            setContentView(R.layout.activity_main);

            buttonMenu1 = (ImageButton) findViewById(R.id.main_start);
            buttonMenu2 = (ImageButton) findViewById(R.id.main_archive);
            buttonMenu3 = (ImageButton) findViewById(R.id.main_help);


            buttonMenu1.setOnClickListener(new View.OnClickListener(){
                public void onClick(View view){
                    Intent intent = new Intent(getApplicationContext(), MapActivity.class);
                    startActivity(intent);
                }
            });
            buttonMenu2.setOnClickListener(new View.OnClickListener(){
                public void onClick(View view){
                    Intent intent = new Intent(getApplicationContext(), StorageActivity.class);
                    startActivity(intent);
                }
            });
        /*
        buttonMenu3.setOnClickListener(new View.OnClickListener(){
            public void onClick(View view){
                Intent intent = new Intent(getApplicationContext(), MapActivity.class);
                startActivity(intent);
            }
        });
        */
        }
}

File: app/src/main/java/com/example/kihunahn/tripdiary/MainActivity.kt
package com.example.kihunahn.tripdiary

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*
import android.content.Intent

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        main_start.setOnClickListener {
            val intent = Intent(this, MapActivity::class.java)
            startActivity(intent)
        }
        /*
        main_archive.setOnClickListener {
            val intent = Intent(applicationContext, StorageActivity::class.java)
            startActivity(intent)
        }
        main_help.setOnClickListener {
            val intent = Intent(applicationContext, MapActivity::class.java)
            startActivity(intent)
        }
        */
    }
}
